core/src/main/scala/kafka/server/AbstractAsyncFetcher.scala

```diff
@@ -166,10 +167,21 @@ abstract class AbstractAsyncFetcher(name: String,
         trace(s"There are no active partitions. Back off for $fetchBackOffMs ms before sending a fetch request")
         true
       case Some(ReplicaFetch(sessionPartitions, fetchRequest)) =>
-        processFetchRequest(sessionPartitions, fetchRequest)
+        processFetchRequestAndMaybeLog(sessionPartitions, fetchRequest)
     }
   }
 
+  private def processFetchRequestAndMaybeLog(sessionPartitions: util.Map[TopicPartition, FetchRequest.PartitionData],
```

Comment:
This method is simply a wrapper on top of processFetchRequest. The method signature is exactly same other than the method name.